craft_archives/repo/package_repository.py

```diff
@@ -144,7 +144,43 @@ class PackageRepository(BaseModel, abc.ABC):
     )
 
     type: Literal["apt"]
-    priority: Optional[PriorityValue] = None
+    """The type of the repository.
+
+    Only APT repositories are supported.
+
+    **Examples**
+
+    .. code-block:: yaml
+
+        type: apt
+
+    """
+
+    priority: Optional[PriorityValue] = Field(
+        default=None,
+        description="The priority of the repository",
+        examples=["always", "999"],
+    )
+    """The priority of the repository.
+
+    This key is used to override the default behavior when picking the source for a
+    package.
```

Comment:
We should strive for present tense and simple verbs, but we're lacking in guidelines for these right now, so I won't insist.

```suggestion
    If set, this key overrides the default behavior when picking the source for a
    package.
```

Reply:
https://github.com/canonical/craft-archives/pull/162/commits/540a69b84954ead70d4c21f736d3a6a484d55f1e